app/views/layouts/_header.html.slim

```diff
@@ -31,13 +31,18 @@ header
                     | #{current_user.tone_name}
                   li
                     = link_to "Profile", current_user
-                  li
-                    = link_to "List Pirates", users_path
-                  li
-                    = link_to "Sign out", destroy_user_session_path, method: :get
                   - if current_user.admin?
                     li
+                      = link_to "Manage Gigs", manage_gigs_path
+                    li
+                      = link_to "Manage Pirates", users_path
+                    li
                       = link_to "Add Pirate", new_user_path
+                  -else
+                    li
+                      = link_to "Manage Pirates", users_path
+                  li
+                    = link_to "Sign Out", destroy_user_session_path, method: :get
```

Comment:
On general principle, I'd try to figure out what was going on with that destroy session link..it's usually put together with a `method: :delete`, on the principle that a GET request should have no side effects--idempotent.